### DISCUSSION TITLE
Slow Search and Loading in custom Docker Network

### DISCUSSION THREAD

**Discussion by lucala** [sct/overseerr#3053] [Help] [answered]
The tiles and search results load extremely slowly on my Overseerr as long as the container is connected via custom docker network. If I switch it to the default docker network it works flawlessly. [I have found this reddit thread but no one knows what the issue is exactly.](https://www.reddit.com/r/Overseerr/comments/pmbgf6/slow_search_results_and_loading_in_general/) - adding DNS entries didn't work for me.

Has anybody had similar issues, or know how I can fix this? I would very much like to have Overseerr remain on the custom network.

**Comment by mduijm** [comment, marked as answer]
Hi,

I had the issue as well with slow response and loading etc. I solved it by adding dedicated DNS lines into my docker-compose file. The software works super fast now:

here my docker-compose details for Overseerr.

```  overseerr:  
    image: lscr.io/linuxserver/overseerr  
    container_name: overseerr  
    environment:  
      - PUID=1000  
      - PGID=1000  
      - TZ=Europe/Amsterdam  
    volumes:  
      - /home/xxxx/docker/overseer:/config  
    ports:  
      - 5055:5055  
    dns:  
      - 1.1.1.1  
      - 8.8.8.8  
    restart: unless-stopped

**Reply by lucala** [reply]
Yes thanks, that solved it. I had entered the DNS as an environment variable which wasn't working, fixed it now.
